docs/timelines/creating-a-timeline.md

```diff
@@ -0,0 +1,103 @@
+
+2. Press \<Record\>, \[Timeline\], then the **Select button** of the fader where you want to store the timeline. The Timeline View window will automatically open.
+
+3. Press the \{O\} (record) button in the top left of the Timeline View window. 
+
+4. Select which track on the timeline you want to record onto using the \[Track\] softkey or tap on the flashing dotted area of the track you want to use. 
+You can add triggers to an existing track or create a new track by tapping on the blank area at the bottom.
+
+    ![Timeline quick record dotted tracks](/docs/images/Timeline-Quickrecord-Dotted.png)
+
+5. Tap again on the track to start recording (or press the \[Start Live Record\] softkey). The workspaces will be outlined in red but recording will not begin until the timecode is started.
+
+6. For internal timecode source, press the \{>\} (play) button in the top left of the timeline window to start the timecode running. If you are using an external timecode source, start it.
+
+7. Now when you make any of the above trigger actions on the console, you will see a trigger recording appear on the timeline. The time range you have recorded is highlighted in red on the timeline view.
+
+8. When you have finished recording press the \{O\} (record) button again and stop the timecode.
+
+You should now see all the triggers you have created laid out on the timeline.
+
+- When using internal timecode, pressing \{O\} (record) then \{>\} (play) will automatically start Live Recording.
+
+- While you are in Live Record, the console will follow the Timeline Release settings as you've programmed them in the [Timeline Options - Release tab](../timelines/timeline-options.md#release-tab), which may make playbacks behave differently to how they are programmed. A warning is displayed in the system prompt area to show you what release settings are in effect.
+
+### Auto Simplify
+
+Normally while you are recording, the \[Auto Simplify\] option is enabled. When you stop the recording, this will simplify your fader movements to a few trigger points with linear fades rather than a large number of points which are difficult to edit. If you really need to capture an exotic fader movement you can turn this option off, but to help keep your timeline easy to edit you should leave it on where you can.
+
+Trigger with multiple points during Live Record:
+
+![Timeline before auto simplify](/docs/images/Timeline-Live-Record.png)
+
+The same trigger after Auto Simplify:
+
+![Timeline after auto simplify](/docs/images/Timeline-Live-Record-Simplified.png)
+
+- When you select a trigger (by clicking on it), there are also context menu options in the \{Tools\} context menu to \{Simplify Selected Triggers\} and \{Smooth Selected Triggers\}. The Smooth option will smoothly fade between selected triggers using the fade time of the following trigger.
```

Comment:
I'm not sure if the difference between simplify & smooth is clear from reading this?

Reply:
That is probably because I am not clear what the smooth option actually does, does it just basically draw a line between selected triggers?

Reply:
Good question. @LawrenceGow ?

Reply:
Smooth will set the fade time so that it fades in a linear fashion to the next trigger. For example:

![image](https://user-images.githubusercontent.com/2736036/103872508-6d27be80-50c6-11eb-83d0-82f554fb742c.png)

- Fade to 100% at 1 second
- Fade to 50% at 2 seconds
- Fade to 0% at 3 seconds

All have no fade times set.

After selecting these triggers and pressing smooth you would get

![image](https://user-images.githubusercontent.com/2736036/103872534-7c0e7100-50c6-11eb-930f-e31f173806d4.png)


- Fade to 100% starting at 1 second and fade time of 1 second
- Fade to 50% starting at 2 seconds over 1 second
- Fade to 0% starting at 3 seconds with no fade time